src/Components/Web.JS/src/Platform/WebView/WebViewStreamingInterop.ts

```diff
@@ -0,0 +1,15 @@
+import { sendJSDataStreamUsingObjectReference } from "../../StreamingInterop";
+
+export async function sendJSDataStreamWebView(data: ArrayBufferView | Blob, streamId: number, chunkSize: number, dotnetStreamReference: any) {
+    async function send(streamId: number, chunkId: number, nextChunkData: Uint8Array | null, error: string | null): Promise<boolean> {
+        if (error) {
+            await dotnetStreamReference.invokeMethodAsync('ReceiveJSDataChunk', streamId, -1, '', error.toString());
+            return false;
+        }
+
+        const nextChunkDataStr = btoa(String.fromCharCode.apply(null, nextChunkData as unknown as number[]));
```

Comment:
Don't we have the ability to pass binary data without base64-encoding it now?

Reply:
I mean, I know the underlying implementation if JS interop for WebView will actually base64 encode it, but that's an internal implementation detail of jsObjectReference.invokeMethodAsync, isn't it?